report/common.py

```diff
@@ -284,7 +310,7 @@ def get_final_target_code(self, benchmark: str, sample: str) -> str:
     targets_dir = os.path.join(self._results_dir, benchmark, 'fixed_targets')
```

Comment:
I reckon this could be the cause.
Previously OSS-Fuzz-Gen saves the final fuzz target in to this dir.
Now it is placed into:
* `results/output-libraw-_zn6libraw14crxdecodeplaneepvj/fuzz_targets/{sample_id}.fuzz_target`
* `results/output-libraw-_zn6libraw14crxdecodeplaneepvj/fuzz_targets/{sample_id}.build_script`